Respect channel list sort in message search (#1237)
GetStream/stream-chat-swiftui: CHANGELOG.md

# Upcoming

### ✅ Added

> - Channel list message search now uses the channel list's sort order: `ChatChannelListViewModel.performMessageSearch()` is set from the channel list query in the default strategy, and is passed to the search API [#1237](https://github.com/GetStream/stream-chat-swiftui/pull/1237)

Reviewer: We can probably make it shorted:
```suggestion
- Channel list message search now uses the channel list's sort order by default [#1237](https://github.com/GetStream/stream-chat-swiftui/pull/1237)
```
